packages/cx-navigation/doc/Menu.md

```diff
@@ -22,27 +25,97 @@ In the end, the menu is rendered in the following structure:
 </Nav>
 ```
 
-The Wrapper is just an `<ul>` element. By using `asBodilessMenu()`, the default MenuClean provides an editable Item list, rendered as `<li>`. The Title inside is just an editable link in the clean component but the base/default tokens provided will replace it with CX MenuTitle.
+The Wrapper is simply an `<ul>` element. By using `asBodilessMenu()`, the default `MenuClean`
+provides an editable Item list, rendered as `<li>`. The Title inside is just an editable link in the
+clean component, but the base/default tokens provided will replace it with Vital MenuTitle.
+
+## Content Editor Details
+
+Where made available by a Site Builder, you can add Menu Items to the Menu. Once added, you can:
+
+- [Add a Menu Title](./MenuTitle#addedit-menu-title);
+- [Add a Menu Link](#add-a-menu-link); and
+- [Add a Sub-Menu](./Sub-Menu#add-a-sub-menu).
+
+![Main Menu Item context menu](../assets/MainMenuItemContextMenu.jpg ':size=50%')
+
+?> **Note:** No (parent) Menu Item with children can be a Link; i.e., if you add a Sub-Menu to a
+Menu Item, the Menu Item cannot then be a Link.
```

Comment:
This is primarly a design choice and not technical choice.   If you want a menu item with subchildren to be a link, we recommend you a submenu indicators so that menu is accessible.    @Guilherme-Almeida-Zeni  is menu indicators in bodiless yes?